spec/spec_helper.rb

```diff
@@ -1,5 +1,5 @@
-require "codeclimate-test-reporter"
-CodeClimate::TestReporter.start
+require 'simplecov'
+SimpleCov.start
```

Comment:
I'm late to this PR, was it failing on master because of this?

Reply:
Yes. It was deprecated in https://github.com/codeclimate/ruby-test-reporter/blob/master/CHANGELOG.md#changes-1